akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -990,7 +990,28 @@ trait FlowOps[+Out, +Mat] {
    * '''Cancels when''' downstream completes
    */
   def groupedWithin(n: Int, d: FiniteDuration): Repr[immutable.Seq[Out]] =
-    via(new GroupedWithin[Out](n, d))
+    via(new GroupedWeightedWithin[Out](n, ConstantFun.oneLong, d).withAttributes(DefaultAttributes.groupedWithin))
```

Comment:
how would I pass the arguments (`n` and `d`)? those are passed now via constructor params...